### ISSUE TITLE
Actionable child is missing from the Actionable view

### ISSUE THREAD

**Issue by SqAtx** [getting-things-gnome/gtg#1081]
From the Actionable view, create the following task
```
parent
- child1
```
The first problem is that child1 doesn't appear in the Actionable view.
Let's go to Open and back to Actionable. child1 appears.
Mark child1 as done by clicking on its checkbox. It disappears and parent appears in its place.
Edit parent. Make it
```
parent
- child1
- child2
``` 
(now child1 has a checked textbox and child2 has an unchecked textbox)
Close the editor.
child2 isn't visible in the Actionable view, _even if you go to a different view and back to Actionable_.

Go to Open and open the parent again.
Its text is now
```
parent

- child1
- child2
``` 
The fact that the text has been modified has already been filed as #1077.
I don't think the blank line is actually related to this. If I remove it, I still can't see child2 in the Actionable view.

I don't know of a way to make child2 appear in the Actionable view without restarting GTG.

**Comment by SqAtx** [comment]
Yeah I just tested in 0.7-dev-20240906 and this specific problem isn't happening. But it could be hiding behind #1039 (which was fixed when I opened this, I think).

I feel like we took steps backwards again. I was testing just now, creating a task just doesn't work. I click on the +, set a title, close the modal, and the task just doesn't get created at all. Creating from the text field works. But that's not always true. When I opened GTG the first time to repro this, creating tasks worked. Super scary.

Let's forget about this specific bug for now.

**Comment by diegogangl** [comment]
Seems like these bugs have been knocked out in Master (`291d49cabb4f4b104e9c9a3eb1adcf7b8e021072`)
